code/modules/hydroponics/grown/garlic.dm

```diff
@@ -0,0 +1,30 @@
+/obj/item/seeds/garlic
```

Comment:
I dont see the mutation for this tbh did you forget it or am I blind?

Reply:
Uhhh, i might have missed it lemme see